applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_base_solver.py

```diff
@@ -519,7 +533,7 @@ def _get_element_condition_replace_settings(self):
                     self.settings["element_replace_settings"]["element_name"].SetString("EulerianConvDiff3D")
                 else:
                     self.settings["element_replace_settings"]["element_name"].SetString("EulerianConvDiff3D8N")
-        elif element_name in ("LaplacianElement","AdjointHeatDiffusionElement"):
+        elif element_name in ("LaplacianElement","AdjointHeatDiffusionElement","QSConvectionDiffusionExplicit","DConvectionDiffusionExplicit"):
```

Comment:
We should find a way to improve this. Note that each new element pollutes the base solver. Can the same behaviour be achieved by introducing an auxiliary function?

Reply:
What comes to my mind is to define `_get_element_condition_replace_settings` inside each solver. This way, we do not pollute the base solver. What do you think?

Reply:
Something of the sort. What we do in the `fluid_solver` is to define a `_ReplaceElementsAndConditions` method.

Reply:
In any case we can do it in a future PR as this would imply changing all the solvers.